test/apiv2/20-containers.at

```diff
@@ -364,11 +372,15 @@ t GET containers/$cid/json 200 \
 t DELETE containers/$cid?v=true 204
 
 # Test Compat Create with default network mode (#10569)
+networkmode=slirp4netns
+if root; then
+    networkmode=bridge
+fi
```

Comment:
This looks like a bug we should also default to bridge as rootless for the compat API, @mheon WDYT?

Reply:
@Luap99 thank you for this. Addressing that is beyond the scope of this PR (and beyond my ability), so I've released the hold. If this situation needs fixing, it should be done in a followup PR.